solutions/usaco-1228.mdx

```diff
@@ -83,4 +84,57 @@ int main() {
 ```
 
 </CPPSection>
+
+<PySection>
+
+```python
+import sys
+t = sys.stdin.readline
+
+def solution():
+
+	n = int(t())
+	
+	g, l, ans = [], [], float('inf')
+	
+	for _ in range(n):
+	
+		a, b = t().split()
+
+		'''
+		The position is stored in g if the cow claims that Bessie is hiding at a location greater than b.
+		The position is stored in l if the cow claims that Bessie is hiding at a location less than b.
+		'''
```

Comment:
1. over 80 chars
2. use double quotes for these multiline string comments

Reply:
am i not already using double quotes?

Reply:
```py
"""
double quotes
"""
```

Reply:
done